src/Common/src/CoreLib/System/Globalization/DateTimeFormatInfo.cs

```diff
@@ -2214,45 +2226,17 @@ internal TokenHashValue[] CreateTokenHashTable()
                 InsertHash(temp, AMDesignator, TokenType.SEP_Am | TokenType.Am, 0);
                 InsertHash(temp, PMDesignator, TokenType.SEP_Pm | TokenType.Pm, 1);
 
-                if (LanguageName.Equals("sq"))
-                {
-                    // Albanian allows time formats like "12:00.PD"
-                    InsertHash(temp, IgnorablePeriod + AMDesignator, TokenType.SEP_Am | TokenType.Am, 0);
-                    InsertHash(temp, IgnorablePeriod + PMDesignator, TokenType.SEP_Pm | TokenType.Pm, 1);
-                }
-
-                // CJK suffix
-                InsertHash(temp, CJKYearSuff, TokenType.SEP_YearSuff, 0);
```

Comment:
>CJKYearSuff [](start = 33, length = 11)

similar to my Hebrew numbers questions. we used to allow parsing some Japanese characters even with Invariant mode and not using Japanese calendar. Now we are disallowing that.